user/languages/clojure.md

```diff
@@ -4,38 +4,64 @@ layout: en
 
 ---
 
-### What This Guide Covers
+<div id="toc">
+</div>
 
-This guide covers build environment and configuration topics specific to Clojure projects. Please make sure to read our [Getting Started](/user/getting-started/) and [general build configuration](/user/customizing-the-build/) guides first.
+<aside markdown="block" class="ataglance">
+
+| Clojure                                     | Default                                   |
+|:--------------------------------------------|:------------------------------------------|
+| [Default `install`](#Dependency-Management) | `project.clj`                             |
+| [Default `script`](#Default-Build-Script)   | `lein test`                               |
+| [Matrix keys](#Build-Matrix)                | `env`, `lein`, `jdk`                      |
+| Support                                     | [Travis CI](mailto:support@travis-ci.com) |
+
+Minimal example:
+
+```yaml
+language: clojure
+```
+
```

Comment:
samesies to C/C++  →  Should we add {: data-file=".travis.yml"} here? And below as needed? I'm thinking maybe this PR pre-dates that option 🤔

Reply:
Yep - AJ did awesome work on ALL the docs, so we just to remember to update open PRs

Reply:
Another reason to get these lang pages merged in sooner rather than later to reduce this kind of thing.